website/docs/cloud-docs/api-docs/variable-sets.mdx

```diff
@@ -687,6 +806,103 @@ curl \
   https://app.terraform.io/api/v2/varsets/varset-kjkN545LH2Sfercv/relationships/workspaces
 ```
 
+## Apply Variable Set to Projects
```

Comment:
How would you feel about moving this section above "Apply Variable Set to Workspaces"? In order of specificity, it feels it make sense to tell folks how to assign to projects _before_ workspaces.

Reply:
That makes sense, I didn't know if there was a preference on ordering.